chia/clvm/spend_sim.py

```diff
@@ -157,7 +158,7 @@ def new_coin_record(self, coin: Coin, coinbase: bool = False) -> CoinRecord:
 
     async def all_non_reward_coins(self) -> List[Coin]:
         coins = set()
-        async with self.mempool_manager.coin_store.db_wrapper.reader_no_transaction() as conn:
+        async with self.coin_store.db_wrapper.reader_no_transaction() as conn:
```

Comment:
you might as well just use `self.db_wrapper` directly here too, right?

Reply:
Yeap.